coconut/cmd/repo_default_revision.go

```diff
@@ -0,0 +1,45 @@
+/*
+ * === This file is part of ALICE O² ===
+ *
+ * Copyright 2019 CERN and copyright holders of ALICE O².
+ * Author: Kostas Alexopoulos <kostas.alexopoulos@cern.ch>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * In applying this license CERN does not waive the privileges and
+ * immunities granted to it by virtue of its status as an
+ * Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/AliceO2Group/Control/coconut/control"
+	"github.com/spf13/cobra"
+)
+
+// repoDefaultRevision sets global and per-repo default revision
+var repoDefaultRevisionCmd = &cobra.Command{
+	Use:   "default-revision",
+	Short: "set default global and per-repository revision",
+	Long: fmt.Sprintf(`The repository default-revision command sets the global default
```

Comment:
I can see that in some commands we use 
```
Long : fmt.Sprintf(`Some text`)
```
while in others we do
```
Long: `Some text`
```

@teo is there any standard or recommended way?

Reply:
@graduta it really depends on what you need to do. Generally we should only wrap in `Sprintf` if we're formatting a string: if you don't have `%s` and `%d` and such, `Sprintf` should be omitted.

The use of plain quotes vs backticks is simply convenience, backticks are a verbatim environment, they make sense for long strings that include line breaks.

Reply:
I changed all repo commands that unnecessarily used `fmt.Sprintf` to "strings".